moduleframework/dockerlinter.py

```diff
@@ -60,7 +60,12 @@ def _get_env(self, value):
 
     def _get_volume(self, value):
         """Function evaluates a value and returns as string."""
-        return get_string(value)
+        try:
+            value = ast.literal_eval(value)
+        except SyntaxError:
+            # Convert string to list
+            return [value]
```

Comment:
is this logic okay?
I don't know what is proper syntax, but I can imagine that there is possible:
```
In [4]:  ast.literal_eval("'abc'")
Out[4]: 'abc'

In [5]:  ast.literal_eval("['abc']")
Out[5]: ['abc']

In [6]:  ast.literal_eval("abc")
ValueError                                Traceback (most recent call last)
...
```
just last one caused error, but in case there will be 'abc' it will not be any error and it returns string.
Is this expected? syntax like ``VOLUME "/etc"`` or ``VOLUME '/etc'`` is not valid?

Reply:
Good point with ValueError. I will add it.